OneCommunityGlobal/HighestGoodNetworkApp
# Alan mark task done dashboard

This is an admin management dashboard task. This PR introduces a new feature to the dashboard page to be able to mark a task as done if you are logged in as an admin. This new feature is under the Tasks and Timelogs component, and in the Tasks tab. Under each task that is just a task and not a subcategory, there will be a check icon next to the bell icon. If you click it a new popup will appear asking you if you want to proceed and confirm that you want to mark the task as done. If you press confirm then it will mark it as done and refresh the tasks.
How to test:

How to test:
- Log in as admin
- Go to the **Tasks and Timelogs** component
- Click on the **Tasks** tab
- Find a task to mark as done
- Click on the check and then confirm
- Wait for the tasks to reload and check if task is gone from list.
